pkg/service/test/test.go

```diff
@@ -509,10 +509,12 @@ func (t *tester) SimulateRequest(cfg *SimulateRequestConfig) {
 		testPass, testResult := t.testHttp(*cfg.Tc, resp, cfg.NoiseConfig, cfg.IgnoreOrdering)
 
 		if !testPass {
+			t.logger.Debug("", zap.Any("matched mocks", cfg.LoadedHooks.GetMatchedMocks()))
```

Comment:
Please add the logic for deleting unused mocks and a flag to enable this feature.

Reply:
will the unused mocks be removed at end of test-set run or test case run ? also we need to rewrite the mock file right ?

Reply:
done!